### PR TITLE
Remove progressbar for WAIC and LOO

### PR DESCRIPTION
**Pull request by AustinRochford**
This hasn't really been useful since the speedup in #2479

### CODE REVIEW

File: pymc3/stats.py [hunk 1]
```diff
@@ -166,17 +166,11 @@ def logp_vals_point(pt):
...
-        logp = (logp_vals_point(pt) for pt in points)
-        return np.stack(logp)
-    finally:
-        if progressbar:
-            points.close()
+    logp = (logp_vals_point(pt) for pt in points)
+    return np.stack(logp)
 
 
-def waic(trace, model=None, pointwise=False, progressbar=False):
```
Comment (aloctavodia) [inline pymc3/stats.py:179]: remove it also from the docstring

File: pymc3/stats.py [hunk 2]
```diff
@@ -166,17 +166,11 @@ def logp_vals_point(pt):
...
+def waic(trace, model=None, pointwise=False):
```
Comment (aloctavodia) [inline pymc3/stats.py]: not related with your PR, but given that you are already there, could you change "widely available information criterion" to "the widely applicable information criterion"? :-)

**Review by aloctavodia — APPROVED** [review]

**Review by aloctavodia — COMMENTED** [review]
remove `tqdm` import

**Comment by aseyboldt** [comment]
We did a release with that kwarg, right? So technically that would be a breaking change...
Also, this executes user code for the logp and depending on the model this can take an arbitrary amount of time, so how about we just change the default to `False`?

**Comment by AustinRochford** [comment]
@aseyboldt that makes sense, i think the default is `False`, so we can leave as-is.

**Pull request closed without merging**